index.html

```diff
@@ -6,22 +6,22 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Home Page</title>
   </head>
-  <body class="bg-color-all font-urbanist">
+  <body class="bg-[#141414] font-urbanist">
     <!-- main container each section in main container -->
     <div class="main-div-container">
       <!-- header and banner -->
       <div class="grid grid-rows-2 gap-0.5">
         <!-- head for home page -->
         <section
-          class="bg-img-for-header text-white grid grid-flow-col justify-items-end items-center p-3"
+          class="bg-img-for-header bg-header-bg bg-no-repeat bg-center bg-cover text-white grid grid-flow-col justify-items-end items-center p-3"
         >
-          <div class="text-size text-color font-medium">
+          <div class="text-[10px] leading-4 text-[#FFFFFF] font-medium">
```

Comment:
I have asked many time to add it to tailwind config file.

Reply:
set in config file bro